### PR TITLE
Add prompt for autostarting service on Windows

### PR DESCRIPTION
**Pull request by kuleshovilya**
Related issue: [3430](https://github.com/microsoft/azure-pipelines-agent/issues/3430)

Description: Add prompt for whether to start configured service on Windows, or keep it stopped, as requested in ticket above

### CODE REVIEW

File: src/Misc/layoutbin/ru-RU/strings.json [hunk 1]
```diff
@@ -538,6 +538,7 @@
     "ServiceInstalled":  "Служба \"{0}\" успешно установлена",
     "ServiceRecoveryOptionSet":  "Для службы \"{0}\" успешно задан вариант восстановления",
     "ServiceStartedSuccessfully":  "Служба \"{0}\" успешно запущена",
+    "StartServiceDescription": "запустить службу после окончания конфигурации? ([Д]а/[Н]ет)",
```
Comment (EzzhevNikita) [inline src/Misc/layoutbin/ru-RU/strings.json:541]: We are adding only English strings, strings for other languages will be translated by the localization team.

Comment (kuleshovilya) [reply src/Misc/layoutbin/ru-RU/strings.json:541]: My bad

**Review by DaniilShmelev — APPROVED** [review]
LGTM
Also, tested these changes manually by configuring an agent. Everything worked as expected.

**Review by EzzhevNikita — APPROVED** [review]

File: src/Agent.Listener/Configuration.Windows/WindowsServiceControlManager.cs [hunk 2]
```diff
@@ -35,7 +35,7 @@ public override void Initialize(IHostContext hostContext)
             _term = HostContext.GetService<ITerminal>();
         }
 
-        public void ConfigureService(AgentSettings settings, CommandSettings command)
+        public void ConfigureService(AgentSettings settings, CommandSettings command, bool preventServiceStart)
```
Comment (anatolybolshakov) [inline src/Agent.Listener/Configuration.Windows/WindowsServiceControlManager.cs:38]: Why do we need to pass preventServiceStart?

Comment (kuleshovilya) [reply src/Agent.Listener/Configuration.Windows/WindowsServiceControlManager.cs:38]: Because of the check on line 135

Comment (anatolybolshakov) [reply src/Agent.Listener/Configuration.Windows/WindowsServiceControlManager.cs:38]: Why can't we retrieve it from command?

Comment (kuleshovilya) [reply src/Agent.Listener/Configuration.Windows/WindowsServiceControlManager.cs:38]: We can, changed it to retrieve

File: src/Misc/layoutbin/en-US/strings.json [hunk 3]
```diff
@@ -182,6 +183,9 @@
     "Run as a Windows service that logs on as a domain account (Windows only)",
     ".\\config.cmd --unattended --url https://myaccount.visualstudio.com --auth pat --token myToken --pool default --agent myAgent --runAsService --windowsLogonAccount myDomain\\myUserName --windowsLogonPassword myPassword",
     "",
+    "Run as a Windows service that logs on as a domain account (Windows only) and doesn't start service immediately",
+    ".\\config.cmd --unattended --url https://myaccount.visualstudio.com --auth pat --token myToken --pool default --agent myAgent --runAsService --windowsLogonAccount myDomain\\myUserName --windowsLogonPassword myPassword",
```
Comment (DaniilShmelev) [inline src/Misc/layoutbin/en-US/strings.json:187]: The `--preventServiceStart` arg is not provided here

Comment (kuleshovilya) [reply src/Misc/layoutbin/en-US/strings.json]: added

**Review by DaniilShmelev — APPROVED** [review]

**Review by anatolybolshakov — APPROVED** [review]
LGTM, thanks!

**Pull request merged**